realm/realm-library/src/androidTest/java/io/realm/TestHelper.java

```diff
@@ -779,7 +779,7 @@ public static void populateForDistinctFieldsOrder(Realm realm, long numberOfBloc
     }
 
     public static void awaitOrFail(CountDownLatch latch) {
-        awaitOrFail(latch, 60);
+        awaitOrFail(latch, 300);
```

Comment:
isn't 300 too long? 60 s is long enough for any test to pass, otherwise there is a bug in the test.